packages/dd-trace/src/appsec/iast/vulnerabilities-formatter/evidence-redaction/sensitive-analyzers/header-sensitive-analyzer.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const {
+  DEFAULT_IAST_REDACTION_NAME_PATTERN,
+  DEFAULT_IAST_REDACTION_VALUE_PATTERN
+} = require('../sensitive-regex')
+
+const { HEADER_NAME_VALUE_SEPARATOR } = require('../../constants')
+
+module.exports = function extractSensitiveRanges (evidence) {
+  const evidenceValue = evidence.value
+  const sections = evidenceValue.split(HEADER_NAME_VALUE_SEPARATOR)
+  const headerName = sections[0]
+  const headerValue = sections.slice(1).join(HEADER_NAME_VALUE_SEPARATOR)
+  if (headerName.match(DEFAULT_IAST_REDACTION_NAME_PATTERN) ||
```

Comment:
Since we are not using the matching result for the regex, we should use `regex.test`